src/components/BlogList.vue

```diff
@@ -130,7 +95,7 @@ watch(
 
 .blog-image {
   width: 100%;
-  height: auto;
+  height: 240px;
```

Comment:
Please read this `maramari` in stackoverflow and determine wether you should use `rem` or `px`